phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -77,6 +79,7 @@ public SQLException newException(SQLExceptionInfo info) {
     MISSING_MAX_LENGTH(207, "22004", "Max length must be specified for type."),
     NONPOSITIVE_MAX_LENGTH(208, "22006", "Max length must have a positive length for type."),
     DECIMAL_PRECISION_OUT_OF_RANGE(209, "22003", "Decimal precision outside of range. Should be within 1 and " + PDataType.MAX_PRECISION + "."),
+    ILLEGAL_OPERATION(210, "22010", "Illegal Operation."),
```

Comment:
Change this to something like this:
CREATE_SCHEMA_NOT_ALLOWED(210, "22010", "Cannot create schema because config " + QueryServicesOptions. IS_NAMESPACE_MAPPING_ENABLED "  + " for enabling name space mapping isn't enabled.")